src/lib/post.ts

```diff
@@ -98,5 +100,26 @@ const getPostByTag = (tag: string, fields: Field[] = POST_FIELDS): Post[] => {
   return posts.filter((post) => post.tags.includes(tag));
 };
 
-export { getPostBySlug, getAllPosts, getPostByTag };
+const randomPostSlug = (excludedSlugs: string[] = []): string | null => {
+  const slugs = getPostSlugs().filter((slug) => !excludedSlugs.includes(slug));
+
+  if (slugs.length === 0) {
+    return null;
+  }
+
+  const index = Math.floor(Math.random() * slugs.length);
+  const slug = slugs[index];
+  excludedSlugs.push(slug);
```

Comment:
Do we need to push the randomized slug here?

Reply:
No, since we no longer return excludedSlugs, it should be removed. Fixed in https://github.com/nimblehq/til-web/pull/33/commits/0a56f2f3b162d6d38fe44642fba0a52fe5a50856